plugins/inputs/windows_event_log/wineventlog/wineventlog.go

```diff
@@ -254,10 +254,6 @@ func (w *windowsEventLog) SetEventOffset(eventOffset uint64) {
 	w.eventOffset = eventOffset
 }
 
-func (w *windowsEventLog) Done(offset state.Range) {
```

Comment:
This was used before the LogEvent had a pointer to the state manager. Now the LogEvent can call the state manager (RangeQueue) directly.